src/main/java/baseball/game/BaseballGame.java

```diff
@@ -0,0 +1,47 @@
+package baseball.game;
+
+import baseball.common.Constants;
+import baseball.game.status.BaseballGameStatus;
+import baseball.player.ComputerPlayer;
+import baseball.player.UserPlayer;
+import baseball.view.InputView;
+import baseball.view.OutputView;
+
+public class BaseballGame {
+    public static BaseballGame instance;
```

Comment:
싱글톤으로 제공하는 필드인데 전역적으로 열려있으면 위험해 보이네요
```suggestion
    private static BaseballGame instance;
```